live/liveview.go

```diff
@@ -806,8 +806,8 @@ func SendInfo(ctx context.Context, info *Info) {
 	if s == nil {
 		return
 	}
-	// TODO should we do this in a goroutine?
-	s.info <- info
+	// don't block when sending info
+	go func() { s.info <- info }()
```

Comment:
I'd rather document that SendInfo blocks, and let the client start it in a goroutine. That provides lots more flexibility.